drools-ansible-rulebook-integration-api/src/main/java/org/drools/ansible/rulebook/integration/api/LogUtil.java

```diff
@@ -10,18 +15,26 @@ private LogUtil() {
 
     // convert java class to python class
     private static Map<Class<?>, String> convertMap = Map.of(
-            java.lang.Integer.class, "int",
-            java.lang.Boolean.class, "bool",
-            java.lang.String.class, "str",
-            java.lang.Double.class, "float",
-            java.util.List.class, "list",
-            java.util.ArrayList.class, "list",
-            java.util.Map.class, "dict",
-            java.util.LinkedHashMap.class, "dict",
-            java.util.HashMap.class, "dict"
+            Integer.class, "int",
+            Boolean.class, "bool",
+            String.class, "str",
+            Double.class, "float",
+            List.class, "list",
+            ArrayList.class, "list",
+            Map.class, "dict",
+            LinkedHashMap.class, "dict",
+            HashMap.class, "dict"
     );
 
     public static String convertJavaClassToPythonClass(Class<?> javaClass) {
-        return convertMap.getOrDefault(javaClass, javaClass.getSimpleName());
+        if (convertMap.containsKey(javaClass)) {
+            return convertMap.get(javaClass);
+        }
+        if (List.class.isAssignableFrom(javaClass)) {
```

Comment:
Should this be `Collection.class` maybe? What about a `Set` for instance?

Reply:
Thanks. Added one more conversion. `Set` to `set`.

If we get other (unexpected) Collection classes which cannot be converted, probably it's better log the class name.